linux_mode/snapshot/recompile_wtf.sh

```diff
@@ -0,0 +1,17 @@
+#!/bin/bash
```

Comment:
I think that if we find out what's going on w/ the registers, we shouldn't need another build script for `wtf` and be able to get rid of this.